test/python/WMCore_t/Configuration_t.py

```diff
@@ -7,10 +7,50 @@
 
 from WMCore.Configuration import ConfigSection
 from WMCore.Configuration import Configuration
+from WMCore.Configuration import ConfigurationEx
 from WMCore.Configuration import loadConfigurationFile
 from WMCore.Configuration import saveConfigurationFile
 
-from WMQuality.TestInitCouchApp import TestInitCouchApp as TestInit
+from WMQuality.TestInit import TestInit
+
+
+class ConfigurationExTest(unittest.TestCase):
+    """
+    test case for Configuration object
+
+    """
+    def setUp(self):
+        """set up"""
+        self.testInit = TestInit(__file__)
+        self.testDir  = self.testInit.generateWorkDir()
+        self.functionSave = "%s/WMCore_Agent_Configuration_t_function.py" % self.testDir
+
+
+    def tearDown(self):
+        """clean up"""
+        self.testInit.delWorkDir()
+
+
+    def testA(self):
```

Comment:
Can you name your test something more descriptive? When it fails, it would be nice to have a description and easily findable string to speed up debugging.